data/json/flags.json

```diff
@@ -2223,5 +2223,10 @@
     "id": "REBREATHER_CART",
     "type": "json_flag",
     "info": "A filter for a rebreather."
+  },
+  {
+    "id": "OLD_GUN",
+    "type": "json_flag",
+    "info": "A flag for older guns, primarily used for an old guns perk for the perks mod right now."
```

Comment:
This shows up in game, so should def be less meta

Reply:
Done